tensordict/functional.py

```diff
@@ -97,7 +97,7 @@ def pad_sequence(
             written.
         device (device compatible type, optional): if provded, the device where the
             TensorDict output will be created.
-        return_mask (bool, optional): if ``True``, a "masks" entry will be returned.
+        return_mask (bool, str): if ``True``, a "masks" entry will be returned. If ``return_mask`` is a string, it will be return the masks and be used as the key for the masks entry.
```

Comment:
```suggestion
        return_mask (bool or NestedKey, optional): if ``True``, a "masks" entry will be returned. If ``return_mask`` is a nested key (string or tuple of strings), it will be return the masks and be used as the key for the masks entry.
```